scripts/import_cloudwatch_log_groups.py

```diff
@@ -0,0 +1,76 @@
+import json
+
+import boto3
+
+from azul import (
+    config,
+    logging,
+)
+from azul.logging import (
+    configure_script_logging,
+)
+from azul.terraform import (
+    terraform,
+)
+
+log = logging.getLogger(__name__)
+
+
+def main():
+    def resource(name):
+        return 'aws_cloudwatch_log_group.' + name
+
+    log_groups = {}
+    tf_component = config.terraform_component
+
+    if tf_component == 'shared':
+        if config.slack_integration:
+            # Chatbot log groups
+            name = resource('chatbot')
+            log_group = '/aws/chatbot/' + config.qualified_resource_name('chatbot')
+            log_groups[name] = log_group
+    else:
+        # Lambda function log groups
+        tf_path = f'{config.project_root}/terraform/api_gateway.tf.json'
+        with open(tf_path, 'r') as f:
+            tf_json = json.load(f)
+        for d1 in tf_json['resource']:
+            for k1, v1 in d1.items():
+                if k1 == 'aws_lambda_function':
+                    for d2 in v1:
+                        for k2, v2 in d2.items():
+                            name = resource(k2 + '_lambda')
+                            log_group = '/aws/lambda/' + v2['function_name']
+                            log_groups[name] = log_group
```

Comment:
Please use our standard variable names: 'resource', 'resource_type`, 'resource_name`.

To simplify the naming, exploit the fact that the TF config is normalized, i.e., that the d1 dicts are singletons.